.github/workflows/check-active-prs-and-slots.yml

```diff
@@ -0,0 +1,101 @@
+name: Check Active PRs and Slots
+
+on:
+  schedule:
+    - cron: "0 12 * * MON" # Schedule to run on every Monday at 12 PM UTC
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: pwsh
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  check-pr-slots:
+    runs-on: ubuntu-latest
+    outputs:
+      SlotIDs: ${{ steps.listOfSlots.outputs.SlotIDs }}
+      AreSlotsEqual: ${{steps.comparision.outputs.AreSlotsEqual }}
+
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v2
+
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v2
+        with:
+          path: ./.github
+
+      - name: Azure CLI - Login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - name: Get list of deployed Slots
+        id: slotList
+        run: |
+          $slots = $(az webapp deployment slot list --name ${{ env.APP_SERVICE_NAME }} --resource-group ${{env.AZURE_RESOURCE_GROUP }} --query '[].name' --output tsv | sed 's/pr-//g')
+          echo "slots=$slots" >> $env:GITHUB_OUTPUT
+
+      - name: Get list of active PRs
+        id: PRList
+        run: |
+          $active_prs=$(curl -s "https://api.github.com/repos/SSWConsulting/SSW.Website/pulls?state=open" | jq -r '.[].number')
```

Comment:
should use the gh cli to do this... something like `gh pr list --state open --json number`

Reply:
Done - but I'm returning a plan list of prs